src/app/session-storage.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+    providedIn: 'root'
+})
+export class SessionStorageService {
```

Comment:
Should move this to /shared, /common or something along those lines. Maybe move the notification service here as well.